src/content/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-advanced-config.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Elasticsearch integration's advanced configuration
+title: Elasticsearch integration's configuration
```

Comment:
i wonder if this needs to be in `""` since the apostrophe may be throwing things off here